spec/index.md

```diff
@@ -2114,6 +2114,24 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `metrics` {#ic-metrics}
+
+:::note
+
+The metrics management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way.
+
+:::
+
+Given a subnet ID as input, this method returns a collection of metrics for the given subnet (field `metrics`) and a timestamp provided in nanoseconds since 1970-01-01 (field `timestamp_nanos`) at which the metrics are sampled.
```

Comment:
```suggestion
Given a subnet ID as input, this method returns a collection of metrics for the given subnet (field `metrics`) and a timestamp provided in nanoseconds since 1970-01-01 (field `timestamp_nanos`) at which the metrics were starting to be sampled.
```
see discussion in comments on design slides https://docs.google.com/presentation/d/1KXIZtYtn658L2mDKrgmL502jeBo1QbHoJRYYAg7Pu68/edit?disco=AAAA4HT7no0.

It is simpler to record the metrics and also accessing a certain days' metrics

Reply:
> were starting to be

this sounds confusing to me; given the comments on the slides, I believe we just sample the metrics at the first block of a calendar day, but this block's timestamp is the timestamp *at* which the metrics are sampled. How this timestamp is established (first/last block of a calendar day, ...) could be stated as a note on the implementation details, but does not seem to change the semantics of the values.

Reply:
Fair point, I probably misinterpreted @derlerd-dfinity's comment on the slide. It is much more clear if it denotes when the counter is sampled. I'll close this thread.